spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/listener/AbstractMessageListenerContainer.java

```diff
@@ -69,6 +69,8 @@
 	private AsyncAcknowledgementResultCallback<T> acknowledgementResultCallback = new AsyncAcknowledgementResultCallback<T>() {
 	};
 
+	private int phase = DEFAULT_PHASE;
```

Comment:
Looking at Spring Kafka I see they have this:
`public static final int DEFAULT_PHASE = Integer.MAX_VALUE - 100; // late phase`

Likely to leave room for users' custom phased components to be phased before and after the containers, so I think we should use it here as well?

Reply:
Makes sense, but this might be a breaking change? People might depend on the current phase of their application components start/stop lifecyle, it might be uncommon though

Reply:
Yup, you're right, it'd be a breaking change.

I'm ok with it as is then, but maybe @maciejwalkowiak would like to weigh in?